CHANGELOG.md

```diff
@@ -3,7 +3,29 @@
 ## Unreleased
 
 ### Breaking changes
-You must make the following changes when you migrate to this release, or your service may break.
+Where applicable, you must make the following changes when you migrate to this release, or your service may break.
+
+#### Check your accordions still display as expected
+
+[See the new design and revised guidance for accordions in the Design System](https://design-system.service.gov.uk/components/accordion/). In particular, review the guidance on when and when not to use accordions.
+
+You should check that your accordions still display as expected if you:
+
+- have applied any custom styles to your accordions
+- use accordions in non-standard ways
+- have section headings that contain unusual formatting (for example, wrappers)
+
+You do not need to make changes to the content that accordions hide or show - this release's changes do not affect it.
+
+These changes were introduced in [pull request #2257: Update accordion design](https://github.com/alphagov/govuk-frontend/pull/2257).
+
+#### Only use the accordion’s summary line for ‘phrasing content’
+
+The summary line is now inside the HTML `button` element, so you can only add ['phrasing content'](https://html.spec.whatwg.org/#phrasing-content) to it. Do not use any non-phrasing content, as the resulting HTML will not be valid.
+
+[Read the accordion guidance to check whether it's the right component to use](https://design-system.service.gov.uk/components/accordion/#when-not-to-use-this-component).
+
+This change was introduced in [pull request #1961: Update accordion guidance for v4.0.0](https://github.com/alphagov/govuk-design-system/pull/1961).
```

Comment:
I kind of understand why we've linked here, but I also wonder if a user would expect this link to take them to the PR where the summary line is moved inside the `button` element (i.e: the govuk-frontend PR). What do you think @hannalaakso ?

Reply:
Thanks both! Which PR is this?

Reply:
@EoinShaughnessy This one: https://github.com/alphagov/govuk-frontend/pull/2257

Reply:
Is this the correct PR to link to? [#2257: Update Accordion design](https://github.com/alphagov/govuk-frontend/pull/2257)

Reply:
Thanks @Hanna!